analyzers/tests/SonarAnalyzer.UnitTest/TestCases/UriShouldNotBeHardcoded.cs

```diff
@@ -74,3 +73,35 @@ void ValidCases(string s)
         }
     }
 }
+
+// https://github.com/SonarSource/sonar-dotnet/issues/7815
+class ReproFN_7815
+{
+    class MyClass
+    {
+        public string FilePath { get; set; }
+    }
+
+    void Method()
+    {
+        var myClass = new MyClass
+        {
+            FilePath = "/my/other/folder" // Compliant - we ignore unix paths by default
+        };
+
+        var myClass2 = new MyClass
+        {
+            FilePath = @"\\my-network-drive\folder\file.txt" // FN
+        };
```

Comment:
I would only leave this example. All the others are FNs for other reason, and when they get fixed they will also start raising in this scenario.